### PR TITLE
Change flex to grid

### PR DESCRIPTION
**Pull request by ivan-kadykalo**
- [DEMO LINK](https://ivan-kadykalo.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/blocks/header.scss [hunk 1]
```diff
@@ -2,31 +2,32 @@
   display: flex;
   align-items: center;
   justify-content: space-between;
-  height: 60px;
-  padding: 0 50px;
-  margin-bottom: 50px;
 
+  height: $header-hight;
```
Comment (VETALEK) [inline src/styles/blocks/header.scss:6]: header-hEight

**Review by VETALEK — APPROVED** [review]
All good

File: src/styles/blocks/catalog.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  justify-content: center;
+  gap: 48px;
+
+  @media (min-width: 1190px) {
```
Comment (illiarerun) [inline src/styles/blocks/catalog.scss:7]: are you sure it should be 1190? check the task again

File: src/styles/blocks/page.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  font-family: Roboto, sans-serif;
+}
+
+.page {
+  margin: 0;
+
+  &__header {
+    margin-bottom: 50px;
+
+  }
```
Comment (illiarerun) [inline src/styles/blocks/page.scss:15]:
```suggestion
    margin-bottom: 50px;
  }
```

File: src/styles/blocks/catalog.scss (hunk 2 above)
Comment (illiarerun) [reply src/styles/blocks/catalog.scss:7]: There should be four columns starting at 1024px
There should be three columns starting at 768px
There should be two columns starting at 488px

**Review by illiarerun — CHANGES_REQUESTED** [review]
Nice start, need to fix a few things, tho

File: src/styles/blocks/catalog.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```
Comment (illiarerun) [inline src/styles/blocks/catalog.scss:3]: use media queries your task still doesn't work properly on resolutions below 1024

**Review by illiarerun — CHANGES_REQUESTED** [review]

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**